packages/components/src/DataTable/DataTable.css

```diff
@@ -0,0 +1,112 @@
+.dataTable {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  border: var(--border-base) solid var(--color-border);
+}
+.dataTable .tableContainer {
+  overflow: auto;
+}
+
+.dataTable .tableContainer table {
+  width: 100%;
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+.dataTable .tableContainer thead {
+  border: none;
+  border-bottom: var(--border-thick) solid var(--color-border);
+  background-color: var(--color-white);
+}
+
+.dataTable .tableContainer .stickyThead {
+  position: sticky;
+  top: -1px;
+  right: 0;
+  left: 0;
+  z-index: 1;
+
+  box-shadow: 0px 1px 0px rgba(63, 63, 68, 0.05),
+    0px 3px 3px rgba(63, 63, 68, 0.15);
+}
+
+.dataTable .tableContainer tbody {
+  position: relative;
+}
+
+.dataTable .tableContainer th {
+  text-align: left;
+  cursor: pointer;
+
+  @media (--handhelds) {
+    padding-right: calc(var(--space-small) + var(--space-smallest));
+    white-space: nowrap;
+  }
+}
+
+.dataTable .tableContainer th:first-child {
+  @media (--handhelds) {
+    position: sticky;
+    left: 0;
+    z-index: 1;
+    height: calc(var(--space-largest) + var(--space-smaller));
+    background-color: var(--color-white);
+  }
+}
+
+.dataTable .tableContainer th:hover,
+.dataTable .tableContainer th:first-child:hover {
+  background-color: var(--color-surface--hover);
+}
+
+.dataTable .tableContainer th > div {
+  padding-left: var(--space-base);
+}
+
+.dataTable .tableContainer tr {
+  position: relative;
+  height: calc(var(--space-largest) + var(--space-smaller));
+  border-top: var(--border-base) solid var(--color-border);
+  border-bottom: var(--border-base) solid var(--color-border);
+}
+
+.dataTable .tableContainer td {
+  padding-left: var(--space-base);
+
+  @media (--handhelds) {
+    padding-right: calc(var(--space-small) + var(--space-smallest));
+    white-space: nowrap;
+  }
+}
+
+.dataTable .tableContainer td:first-child {
+  font-weight: bold;
+
+  @media (--handhelds) {
+    position: sticky;
+    left: 0;
+    z-index: 1;
+    height: calc(var(--space-largest) + var(--space-smaller));
+    background-color: var(--color-white);
+  }
+}
+
+.dataTable .tableContainer th:first-child::after,
+.dataTable .tableContainer td:first-child::after,
+.dataTable .tableContainer th:first-child::before,
+.dataTable .tableContainer td:first-child::before {
+  @media (--handhelds) {
+    content: "";
+    display: inline-block;
+    position: absolute;
+    top: 0px;
+    right: 0px;
+    width: var(--space-minuscule);
+    height: 100%;
+
+    box-shadow: 1px 0px 0px rgba(63, 63, 68, 0.05),
```

Comment:
Looks like this shadow might benefit from some adjustment - it looks like it rounds off at the edges of the top and bottom of each cell:
![image](https://user-images.githubusercontent.com/39704901/196562829-1427c889-dff2-4de8-a3ef-d0f9aa13dccb.png)

Reply:
That's a good catch, thanks a lot @chris-at-jobber for picking this up we have focused more on the functionality side of things, We should massage this CSS code a bit for sure. 

About the build, we should have a working deployment now that some fixes pushed to `master` were merged here.

We should be adding better examples that would show all the capabilities of this component as well as some code refactoring to address the many points discussed in this PR.

thank you guys for the support so far 😄